### PR TITLE
add credProps to example

### PR DESCRIPTION
**Pull request by timcappalli**
#226 add credProps to create example

### CODE REVIEW

File: content/en/docs/use-cases/bootstrapping/index.md [hunk 1]
```diff
@@ -168,6 +168,9 @@ navigator.credentials.create({
       residentKey: "required",
       userVerification: "preferred"
     },
+    "extensions": {
+      "credProps": true
+    }
```
Comment (MasterKale) [inline content/en/docs/use-cases/bootstrapping/index.md:173]: This seems like an arbitrary addition without some explanatory text. Will something get added to explain why we recommend using this extension, and how to use its return value?

Comment (timcappalli) [reply content/en/docs/use-cases/bootstrapping/index.md:173]: we currently don't explain any of them. That is probably something we need to add more globally at a later date.

Comment (MasterKale) [reply content/en/docs/use-cases/bootstrapping/index.md:173]: What motived adding this, though? Not even #226 has context. If this is a "shut up Matt and approve" then lmk.

Comment (timcappalli) [reply content/en/docs/use-cases/bootstrapping/index.md:173]: I noticed it was missing :)

**Review by MasterKale — APPROVED** [review]

**Pull request merged**